src/components/patterns/link.tsx

```diff
@@ -44,10 +44,14 @@ export const DrupalActionLink = ({href, children, ...props}: PropsWithChildren<{
     <Link
       href={href}
       {...props}
-      className={twMerge("relative pr-30 hocus:text-brick no-underline rs-mt-neg1 pt-10", props.className)}
+      className={'pt-10'}
     >
-      {children}
-      <ChevronRightIcon className="inline absolute top-0 right-0 h-full pt-10"/>
+      <tspan  className={twMerge("relative pr-30 no-underline rs-mt-neg1 hover:bg-black-10 hover:text-brick hocus:underline focus:bg-black-true active:bg-black-true", props.className)}
```

Comment:
why different style for hover vs focus?

Reply:
@pookmish I brought that over from other link styles while trying to figure this one out. 

You know, I had a feeling that I should have reviewed this before assigning it you. I should have listened to that thought.

Reply:
My apologies.